### DISCUSSION TITLE
`<breadcrumb>`

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#116] [State of HTML Features]
### Specification URL:

https://open-ui.org/components/breadcrumb/

### How stable is this feature?

Future (No implementations yet)

### Type of feature

HTML element

### What focus area does this feature fall into?

Semantic HTML

**Comment by domenic** [comment]
It looks like the proposed HTML already exists, and this is not an actual new element proposal? It's just saying (like the HTML Standard [already does](https://html.spec.whatwg.org/#rel-up)) that nav is the appropriate wrapper element for breadcrumbs?

I admit the OpenUI page makes things confusing by using tag name syntax like `<breadcrumb>`, `<ordered-list>`, and `<list-item>`, but the actual code example uses the normal HTML elements nav/ol/li.

So I'm unclear what a state of HTML survery question about this would be asking? Is it like a knowledge-quiz thing, "did you know that nav elements are used for breadcrumb navigation?"
